go/libkb/login_session.go

```diff
@@ -38,11 +38,20 @@ func NewLoginSession(emailOrUsername string, g *GlobalContext) *LoginSession {
 func NewLoginSessionWithSalt(emailOrUsername string, salt []byte, g *GlobalContext) *LoginSession {
 	ls := NewLoginSession(emailOrUsername, g)
 	ls.salt = salt
+	// XXX are these right?  is this just so the salt can be retrieved?
 	ls.loaded = true
 	ls.cleared = true
 	return ls
 }
 
+func (s *LoginSession) Status() string {
```

Comment:
A struct for the status is going to end up just copying half the fields of this type, but ok...